pkg/client/interface.go

```diff
@@ -274,4 +274,18 @@ type SessionQueryClient interface {
 		serviceId string,
 		blockHeight int64,
 	) (*sessiontypes.Session, error)
+	// GetParams queries the chain for the current session module parameters.
+	GetParams(ctx context.Context) (*sessiontypes.Params, error)
+	// GetSessionGracePeriodEndHeight returns the height at which the grace period for
+	// the session ending with sessionEndHeight elapses.
+	GetSessionGracePeriodEndHeight(ctx context.Context, sessionEndHeight int64) (int64, error)
+	// GetSessionGracePeriodBlockCount returns the number of blocks in the grace period
+	// for the session which includes queryHeight.
+	GetSessionGracePeriodBlockCount(ctx context.Context, queryHeight int64) (uint64, error)
```

Comment:
Is there a way to just have one o these?

Reply:
As per https://github.com/pokt-network/poktroll/pull/538#issuecomment-2126369870, `#GetSessionGracePeriodBlockCount()` will go away (for now). Given that, I think `#GetSessionGracePeriodEndHeight()` also can continue to be a static function, instead of becoming a client method.